assets/src/edit-story/karma/textStyle.cuj.karma.js

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { waitForElementToBeRemoved } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import useInsertElement from '../components/canvas/useInsertElement';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../app/font/defaultFonts';
+import { useStory } from '../app/story';
+import { Fixture } from './fixture';
+
+describe('Element: Text', () => {
+  let fixture;
+
+  const addText = async (extraProps = null) => {
+    const insertElement = await fixture.renderHook(() => useInsertElement());
+    await fixture.act(() =>
+      insertElement('text', {
+        font: TEXT_ELEMENT_DEFAULT_FONT,
+        content: 'hello world!',
+        x: 40,
+        y: 40,
+        width: 250,
+        ...extraProps,
+      })
+    );
+  };
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+    await addText();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  describe('CUJ: Creator Can Style Text: Use font picker', () => {
+    // There are 3 curated fonts included by default even though the total is more.
+    const DEFAULT_VISIBLE_FONTS = 3;
+    // Timeout used for submitting / search update + 50ms (250 + 50).
+    const TIMEOUT = 300;
+    const openFontPicker = async () => {
+      const input = await fixture.screen.getByLabelText('Edit: Font family');
```

Comment:
I'd like to see all these `getBy*` moved to the fake DOM structure, but we can do that in a follow-up ticket (please create one for it) - let's just get this merged as is!

Reply:
New issue: https://github.com/google/web-stories-wp/issues/3866